lambdas/webhook_processor/README.md

```diff
@@ -0,0 +1,23 @@
+# Webhook Processor
+
+Processes webhooks by reading them from a kinesis stream and forwarding to Jenkins
+
+To zip and upload a new version, using the aws cli:
+```
+zip webhook_processor.zip webhook_processor.py
+aws s3 cp webhook_processor.zip s3://edx-testeng-lambda-webhook/
+rm webhook_processor.zip
+```
+
+Now you need to update (and apply) the terraform for the lambda so that it will use
+the new version.
+Unfortunately finding the version number is not possbile via the aws cli,
+only through the REST API, the SDKs, or the console.
+Go to the s3 bucket web interface (https://console.aws.amazon.com/s3)
+Navigate to the bucket, and then the .zip file.
+Look for the "Version ID" on the Overview tab.
+
+This is what you will put in the terraform.tfvars file.
+  lambda2_version = "<the new version id>"
+
+Note the target (jenkins) URL is also configured in the terraform.tfvars file.
```

Comment:
How do we plan on modifying the target in the future? Will we need to reapply the terraform each time or will it be a available via an environment variable?